rdr_service/alembic/versions/058fde8817a7_adding_site_name_to_w1_raw.py

```diff
@@ -0,0 +1,61 @@
+"""adding_site_name_to_w1_raw
+
+Revision ID: 058fde8817a7
+Revises: 7dc15d7b7284, e249c8cb27a9, 5d4e56e61d6f
+Create Date: 2021-07-22 13:24:18.553753
+
```

Comment:
Can you prefix this PR title with "MIGRATION"?

Reply:
Done